action/stakerestake_test.go

```diff
@@ -37,7 +32,7 @@ func TestRestake(t *testing.T) {
 	require.Equal(uint64(10700), gas)
 	cost, err := stake.Cost()
 	require.NoError(err)
-	require.Equal("107000", cost.Text(10))
+	require.Equal("0", cost.Text(10))
```

Comment:
does this mean stake.Cost() change to 0 now? is Cost() used in calculating gas charge?